server/cluster/cluster.go

```diff
@@ -159,10 +166,12 @@ func (c *Cluster) CreateTable(ctx context.Context, nodeName string, schemaName s
 		return CreateTableResult{}, errors.WithMessage(err, "topology manager add table")
 	}
 
-	return CreateTableResult{
+	ret := CreateTableResult{
 		Table:              table,
 		ShardVersionUpdate: result,
-	}, nil
+	}
+	log.Info(fmt.Sprintf("create table succeed, result:%v", ret), zap.String("cluster", c.Name()))
```

Comment:
It seems strange to use two output methods at the same time, maybe it is better to use the same.